docs/src/test/scala/docs/http/scaladsl/ServerSentEventsExampleSpec.scala

```diff
@@ -0,0 +1,44 @@
+package docs.http.scaladsl
```

Comment:
needs copy header :)
yeah we should use your plugin for it, but had some trouble with it back then ;)